app/src/main/java/com/example/concordia_campus_guide/Models/Rooms.java

```diff
@@ -15,4 +16,12 @@ public Rooms(List<RoomModel> rooms) {
     public List<RoomModel> getRooms() {
         return Rooms;
     }
+
+    public List<Place> getPlaces(){
+        List<Place> toReturn = new ArrayList<Place>();
```

Comment:
Here too, var name could be changed to "rooms"

Reply:
done